src/Query/UdpTransportExecutor.php

```diff
@@ -148,8 +148,8 @@ public function query(Query $query)
             // This can also be reproduced by writing data exceeding `stream_set_chunk_size()` to a server refusing UDP data.
             // fwrite(): send of 8192 bytes failed with errno=111 Connection refused
             \preg_match('/errno=(\d+) (.+)/', $error, $m);
-            $errno = isset($m[1]) ? (int) $m[1] : 0;
-            $errstr = isset($m[2]) ? $m[2] : $error;
+            $errno = (int) $m[1] ?? 0;
```

Comment:
Casting operator precedence requires additional parenthesis:

```suggestion
            $errno = (int) ($m[1] ?? 0);
```

Reply:
Thanks for the hint, fixed it :+1: